config/runtime.exs

```diff
@@ -27,3 +27,18 @@ config :realtime_signs, RealtimeSigns.Scheduler, jobs: scheduler_jobs
 if config_env() == :prod do
   config :realtime_signs, RealtimeSignsWeb.Endpoint, secret_key_base: System.fetch_env!("SECRET_KEY_BASE")
 end
+
+# For maintaining backwards compatibility with opstech3
+splunk_token = System.get_env("PROD_SIGNS_SPLUNK_TOKEN", "")
+
+if config_env() == :prod and splunk_token != "" do
+  config :logger, backends: [Logger.Backend.Splunk, :console]
+
+  config :logger, :splunk,
+  connector: Logger.Backend.Splunk.Output.Http,
+  host: 'https://http-inputs-mbta.splunkcloud.com/services/collector/event',
+  token: splunk_token,
+  format: "$dateT$time [$level]$levelpad node=$node $metadata$message\n",
```

Comment:
question: is the `:console` logger configured to use the format as well?

Reply:
Missed that, thank you for the catch!